src/configure/configure.ts

```diff
@@ -145,6 +147,7 @@ class Orchestrator {
                     this.inputs.pipelineConfiguration.template = shortlistedTemplates[0];
                 }
                 else {
+                    telemetryHelper.logError(Layer, TracePoints.SelectTemplate, new Error(Messages.TemplateNotFound));
```

Comment:
Shouldn't the trace point be something like `TemplateNotFound`. This name does not sound very intuitive when querying for such failures.

Reply:
Also, should we add more details in the error message to tell us what is the scenario where we could not find any template. Should we log the repo id in the message. Or should we log the number of applicationSettings present in repo analysis result. These can help us diagnose what are the cases when no templates are being found.

Reply:
> Shouldn't the trace point be something like TemplateNotFound. This name does not sound very intuitive when querying for such failures.

Or is the idea to have a trace point for each action, and then use that trace points for all errors while executing that action. If so, this sounds fine.

Reply:
yeah, another trace point will be a better idea. Also, will append repoId with the message.